mediaplayer.py

```diff
@@ -163,3 +163,5 @@ def SetPosition(self, TrackId, Position):
     @dbus.service.method("org.mpris.MediaPlayer2.Player", in_signature="s")
     def OpenUri(self, URI):
         pass
+
+# vim: set tabstop=4 expandtab
```

Comment:
I don't know much about using vim, is there a way that this can get moved to some config file?

Reply:
Modelines can be used by other editors too, I'm using xed.

I suppose I could also use an .editorconfig instead. Will make a commit.

Reply:
Pushed a new commit for an editorconfig.